charts/ztka/templates/_helpers.tpl

```diff
@@ -213,3 +221,16 @@ Get user full-qualified domain.
 {{- define "ztka.userFQDN" -}}
 {{.Values.fqdn.userSubdomain}}.{{.Values.fqdn.domain}}
 {{- end -}}
+
+{{/*
+Get paralus service with port that is endpoint for kratos after login webhook.
+*/}}
+{{- define "ztka.afterLoginWebhookWithPort" -}}
+{{ $url := "http://a.aaa"}}
+{{- range .Values.services.paralus.ports -}}
+  {{- if eq .name "http" -}}
```

Comment:
what about https ?

Reply:
There are at present 3 identifiers:
  ```
  - name: http
    containerPort: 11000
  - name: rpc
    containerPort: 10000
  - name: relay-peering
    containerPort: 10001
```
Using only the first